### PR TITLE
fix(frontend): Sidebar scroll bb-277

### PR DESCRIPTION
**Pull request by ihorLysak**
### Summary

Scrolling sidebar issue was caused by QuizForm component which was present at the bottom at the app component. I suppose it was left there because otherwise the code wouldn't pass the lint stage, since QuizForm is not used in a meaningful way right now. I suppose this component is needed for [onboarding/quiz flow](https://github.com/orgs/BinaryStudioAcademy/projects/29/views/1?filterQuery=quiz&pane=issue&itemId=77655669), so i decided to add a temporary display:none to it until the mentioned task is implemented.

https://github.com/user-attachments/assets/bce04186-406d-4821-8ad4-540c841587fa

### CODE REVIEW

File: apps/frontend/src/libs/components/app/app.tsx [hunk 1]
```diff
@@ -41,9 +41,9 @@ const App: React.FC = () => {
 			</div>
 
 			{isRoot && (
-				<>
+				<div style={{ display: "none" }}>
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/app/app.tsx:44]: Not a proper solution. Move QuizForm to Root screen instead

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by marharita08 — APPROVED** [review]

File: apps/frontend/src/index.tsx [hunk 2]
```diff
@@ -25,7 +26,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										component="Root"
+										component={<QuizForm />}
```
Comment (fshabanov) [inline apps/frontend/src/index.tsx:29]: This shouldn't be in ROOT path. QuizForm will be removed in another ticket, which is in progress
Did you test the case, where the main content is too big?

Comment (ihorLysak) [reply apps/frontend/src/index.tsx]: https://github.com/user-attachments/assets/346c9b76-3ca9-4e40-8a32-e4bf8f7ed5cf

**Comment by fshabanov** [comment]
The changes were done as part of #263. This PR introduces no new changes - closing it

**Pull request closed without merging**